src/service/ops/resurface.js

```diff
@@ -72,7 +72,8 @@ async function before(data) {
 }
 
 async function after(data) {
-  return require('./pool').after(data)
+  // return require('./pool').after(data)
```

Comment:
Forgot to remove this?

Reply:
The resurface operation is commented. Raising warning and email alert instead.